### PR TITLE
Add BigQuery handler

### PR DESCRIPTION
**Pull request by ZoranPandovski**
This PR adds the initial version of the Google BigQuery handler. 
> Note we need more tests

### CODE REVIEW

File: mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        """
+        Receive SQL query and runs it
+        :param query: The SQL query to run in BigQuery
+        :return: returns the records from the current recordset
+        """
+        client = self.connect()
+        try:
+            query = client.query(query)
+            result = query.to_dataframe()
+            print('INNNNN', result)
```
Comment (StpMax) [inline mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py:86]: todel

File: mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            )
+        return response
+
+    def query(self, query: ASTNode) -> Response:
+        """
+        Retrieve the data from the SQL statement with eliminated rows that dont satisfy the WHERE condition
+        """
+        renderer = SqlalchemyRender(BigQueryDialect)
+        query_str = renderer.get_string(query, with_failback=True)
+        print('QUERYYY ', query_str)
```
Comment (StpMax) [inline mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py:108]: todel

File: mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+from mindsdb_sql.render.sqlalchemy_render import SqlalchemyRender
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb_sql.parser.ast.base import ASTNode
+from collections import OrderedDict
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+from sqlalchemy_bigquery.base import BigQueryDialect
```
Comment (StpMax) [inline mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py:12]: should it be in requirements?

File: mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+        query_str = renderer.get_string(query, with_failback=True)
+        print('QUERYYY ', query_str)
+        return self.native_query(query_str)
+
+    def get_tables(self, dataset_id) -> Response:
+        """
+        Get a list with all of the tabels in BigQuery
+        """
+        client = self.connect()
+        result = client.list_tables(dataset_id)
```
Comment (StpMax) [inline mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py:116]: response should be formatted as 'Response'

File: mindsdb/integrations/handlers/bigquery_handler/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+google-cloud-bigquery
+db-types
```
Comment (StpMax) [inline mindsdb/integrations/handlers/bigquery_handler/requirements.txt:2]: should it be?

File: mindsdb/integrations/handlers/bigquery_handler/tests/test_bigquery_handler.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import unittest
+from mindsdb.integrations.handlers.bigquery_handler.bigquery_handler import BigQueryHandler
+
+
+class PostgresHandlerTest(unittest.TestCase):
```
Comment (StpMax) [inline mindsdb/integrations/handlers/bigquery_handler/tests/test_bigquery_handler.py:5]: BigQueryHandlerTest :)

**Review by StpMax — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py (hunk 3 above)
Comment (ZoranPandovski) [reply mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py:12]: Yes, adding it

**Review by StpMax — APPROVED** [review]

**Pull request merged**